src/components/PromptSection/PromptSection.js

```diff
@@ -38,8 +39,18 @@ class PromptSection extends Component {
     return Math.floor(Math.random() * this.state.promptsData.length);
   }
 
-  saveFavorite = () => {  
-    //This needs to the a post request that adds this.state.randomPrompt to the favorites table
+  saveFavorite = () => {
```

Comment:
I really liked the way that you used the index property to send all the messages !!